book/src/themes.md

```diff
@@ -1,38 +1,62 @@
 # Themes
 
-To use a theme add `theme = "<name>"` to your [`config.toml`](./configuration.md) at the very top of the file before the first section or select it during runtime using `:theme <name>`.
+To use a theme add `theme = "<name>"` to the top of your [`config.toml`](./configuration.md) file, or select it during runtime using `:theme <name>`.
 
-## Creating a theme
+## Creating a Theme
 
-Create a file with the name of your theme as file name (i.e `mytheme.toml`) and place it in your `themes` directory (i.e `~/.config/helix/themes`). The directory might have to be created beforehand.
+### Creating Your Theme File
 
-The names "default" and "base16_default" are reserved for the builtin themes and cannot be overridden by user defined themes.
+To create a theme file:
 
-The default theme.toml can be found [here](https://github.com/helix-editor/helix/blob/master/theme.toml), and user submitted themes [here](https://github.com/helix-editor/helix/blob/master/runtime/themes). 
+1. Create a 'themes' folder in your user configuration folder (e.g.
+   `~/.config/helix/themes`).
```

Comment:
I actually wasted a few minutes figuring this out myself because I do have a pre-existing .config folder in my home directory, but it isn't where helix looks for themes. 
```suggestion
   `~/.config/helix/themes`, or `%AppData%\helix\themes` on Windows).
```

Reply:
Cheers!